tree/30_generic_methods/file_ensure_lines_present.cf

```diff
@@ -28,6 +28,11 @@
 
 bundle agent file_ensure_lines_present(file, lines)
 {
+  vars:
+      "old_class_prefix" string => canonify("file_ensure_lines_present_${file}");
+      "class_prefix"     string => canonify(join("_", "this.callers_promisers"));
+
   methods:
-      "wrapper" usebundle  => file_enforce_content("${file}", "${lines}", "false");
+      "enforce lines content" usebundle => file_enforce_content("${file}", "${lines}", "false");
+      "new result classes"    usebundle => _classes_copy("${class_prefix}_enforce_lines_content", "${class_prefix}");
```

Comment:
If you don't have the stack class_prefix will be empty. _classes_copy detects this and doesn't do anything.

Indeed, old_class_prefix can be removed